src/utils/motions.ts

```diff
@@ -53,3 +62,34 @@ export const writeMintTokensMotionToDB = async (
     },
   });
 };
+
+const getMotionListenerRemovers = (
```

Comment:
Should they be motion specific, or contract address specific? If an extension is uninstalled, a colony deleted (for example, can't happen right now), etc., we probably want to remove all the listeners for that address, not just motions?
This would be another good reason to extract the entire listener removers into their own little module